src/main/router-content-types.ts

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
I think that all of these router files could be in there own folder:

```
src/
  main/
    router/
      content-types.ts
      router.ts
      router.injectable.ts
      ...
```

Reply:
Done